index.html

```diff
@@ -8,27 +8,166 @@
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins:wght@300;400;500&display=swap" rel="stylesheet"> 
     <link rel="stylesheet" href="./CSS/styles.css">
-    <title>Portfolio Project</title>
+    <title>Portfolio Projec</title>
     <link rel="icon" href="favicon.ico" type="image/x-icon">
 </head>
 <body>
-<header>
     <nav class="nav">
         <div class="menu">
-            <a href="#"><img src="Images/menu.png" alt="menu"></a>
+            <a href="#"><img src="Images/menu.png" alt="My Menu"></a>
         </div>
     </nav>
-</header>
 
     <section class="headline">
         <div class="headtitle">
-            <h1 class="htitle" >Muhammad Ashraf</h1>
+            <h1 class="mttitle" >Muhammad Ashraf</h1>
         </div>
-        <div class="htext">
-            <p class="htext" >Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
+        <div class="maintxt">
+            <p class="mtext" >Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a Project you need coded, don’t hesitate and contact me.</p>
         </div>
         <button type="button" class="colbtn">Start Collaboration</button>
-        <div class="dcmg">
+        <div class="decoimg">
+        </div>
+    </section>
+
+    <section class="works">
+        <div class="headtitle">
+            <h1 class="mttitle">Projects</h1>
```

Comment:
[OPTIONAL] It is recommended to have only one h1 for each page, because you use one before, use h2 here